compose/mpp/demo/src/commonMain/kotlin/androidx/compose/mpp/demo/App.kt

```diff
@@ -33,6 +33,8 @@ val MainScreen = Screen.Selection(
     LazyLayouts,
     TextFields,
     AndroidTextFieldSamples,
+    Screen.Example("BottomSheetScaffold") { BottomSheetScaffoldExample() },
+    Screen.Example("ModalNavigationDrawer") { ModalNavigationDrawerExample() }
```

Comment:
Does it release require two new separate pages? Can we combine it or add to existing ones?

Reply:
I wanted to, but

- They're both fullscreen elements, so combining them in one page is extra work.
- I couldn't immediately think of a common page to put them in.

Since it's not really important, I just went with two pages.

Reply:
I had the same issue with SearchBar (ended with Example1 page). Let's introduce one more sub-list page like textfields, bugs, etc and move all material elements to that sub-llist